include/seqan3/core/platform.hpp

```diff
@@ -332,6 +332,15 @@
 #   endif
 #endif
 
+//!\brief See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100139
+#ifndef SEQAN3_WORKAROUND_GCC_100139 // not yet fixed
```

Comment:
```suggestion
//!\brief See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100139
//!       std::views::{take, drop} do not type-erase. This is a defect within the standard lib.
#ifndef SEQAN3_WORKAROUND_GCC_100139 // not yet fixed
```

We don't do this for every workaround, but I would be in favor of adding a description to the workaround briefs.

Reply:
open